tests/test/baseledger_abi.json

```diff
@@ -0,0 +1,443 @@
+[
```

Comment:
did not add mechanism to re-generate abi since it is not so important, we won't have to change it most likely, at least not often, so i just generated it and copied here
maybe we can just add abi exporter package that i used here, to baseledger contracts repo, so this is at least easier in future?

Reply:
i am ok with manual as contract changes are going to be rare